lib/cogctl/actions/groups/create.ex

```diff
@@ -2,7 +2,6 @@ defmodule Cogctl.Actions.Groups.Create do
   use Cogctl.Action, "groups create"
 
   alias Cogctl.Actions.Groups
-  alias CogApi.HTTP.Client
```

Comment:
This was just a matter of being consistent across the different files. Not a necessary change.